cmd/create.go

```diff
@@ -54,7 +54,7 @@ kn func create
 # Create a Quarkus function project in the directory "sample-service". 
 # The directory will be created in the local directory if non-existent and 
 # the project is called "sample-service"
-kn func create --runtime quarkus myfunc
+kn func create --runtime quarkus sample-service
 
 # Create a function project that uses a CloudEvent based function signature
 kn func create --template events myfunc
```

Comment:
Should this also be changed?

Reply:
not necessarily.. but I agree with you it would get better for sample purposes.